src/components/DatatableV2/Datatable.types.ts

```diff
@@ -110,10 +110,6 @@ export type DatatableColumnDef<D, V = unknown> = Omit<
    * Enables/disables actions menu in header for given column.
    */
   enableColumnActions?: boolean;
-  /**
-   * Enables/disables ordering for given column.
-   */
-  enableOrdering?: boolean;
```

Comment:
Please add `enableOrdering` to the `Omit<ColumnDef<D, V>, ...>` on the line 20 to prevent exposing it through bast react-table types.